src/vaadin-grid-data-provider-mixin.d.ts

```diff
@@ -51,7 +51,7 @@ interface DataProviderMixin {
    *     are requested, total number of items in the requested sublevel.
    *     Optional when tree is not used, required for tree.
    */
-  dataProvider: GridDataProvider | null | undefined;
+  dataProvider: GridDataProvider<any> | null | undefined;
```

Comment:
Should not we try to use `unknown` instead of `any` to force the developer to explicitly specify the type of the data in the grid?